components/flatEditor.js

```diff
@@ -647,14 +647,18 @@ function FlatEditor({
                 setRefId(score.scoreId);
               })
               .catch((e) => {
-                e.message = `flat error: ${e.message}, not loaded from scoreId, score: ${JSON.stringify(score)}, orig: ${orig}, colors: ${colors}`;
-                if (debugMsg){
```

Comment:
ok, but like, can you still keep `debugMsg` when it's available at least in the console error. the point of the hasty change was to be able to distinguish which instances of the Flateditor are experiencing the errors?

Reply:
Admittedly, my commenting out was hasty, especially considering the latter of the runtime errors seems to only happen when the scores lock. uncommented now